pnc-model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -57,10 +59,13 @@
 
     private String patchesUrl;
 
+    @NotNull
     private Timestamp startTime;
 
+    @NotNull
     private Timestamp endTime;
 
+    @NotNull
     @ManyToOne
     private User user;
 
```

Comment:
Did you mean the "user" field?  I think even with auditing of changes to BuildConfiguration, we'd still need to record the "user" in the build record to know who triggered the build.